packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -122,8 +145,12 @@ export class Persona extends React.Component<IPersonaProps, any> {
       <div { ...divProps } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
         { size !== PersonaSize.tiny && (
           <div className='ms-Persona-imageArea'>
-            <div className={ css('ms-Persona-initials', PERSONA_INITIALS_COLOR[initialsColor]) }>{ imageInitials }</div>
-            <Image className='ms-Persona-image' imageFit={ ImageFit.cover } src={ imageUrl } shouldFadeIn={ imageShouldFadeIn } />
+            {
+              !this.state.isImageLoaded &&
```

Comment:
it seems that if the image is fading in, this would create a bit of a janky experience. E.g. it would be blue, then on fade in, go white background for a second while fading in.